### ISSUE TITLE
compilation error in Windows

### ISSUE THREAD

**Issue by zamir1001** [oneapi-src/oneDNN#1449]
Hi,

I get a compilation error in jit_uni_reorder_utils.cpp on line 165 (latest commit https://github.com/oneapi-src/oneDNN/commit/3c0655cc27e6289bdec5381043250a32b40151cb):

oneDNN\src\cpu\x64\jit_uni_reorder_utils.cpp(165,1): error C2220: the following warning is treated as an error
oneDNN\src\cpu\x64\jit_uni_reorder_utils.cpp(165,1): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
oneDNN\src\cpu\x64\jit_uni_reorder_utils.cpp(165,1): warning C4267: 'initializing': conversion from 'size_t' to 'const int', possible loss of data

```
static void prb_set_compensation_strides(prb_t &p) {

    auto get_next_parent_node = [&](int cur_node) -> int {
        const int cur_id = p.nodes[cur_node].dim_id;
        for (int d = cur_node + 1; d < p.ndims; ++d) {
            if (p.nodes[d].dim_id == cur_id) return d;
        }
        return -1;
    };

    auto require_n_stride = [&](int cur_node) -> bool {
        const int parent = get_next_parent_node(cur_node);
        if (parent < 0) return false;

==>     const int p_n = p.nodes[parent].n;

        // if 'parent_node.n' is larger than 1, then cur_node stride
        // is 'cur_node.n'
        return p_n > 1;
    };
```

thanks

**Comment by igorsafo** [comment]
Hi @zamir1001 , thanks for reporting the issue.

The issue is fixed in f53b80a .

Regards,
Igor

**Comment by zamir1001** [comment]
Hi,

thanks for the quick fix!
